backstop/scenarios/tile.js

```diff
@@ -18,104 +18,50 @@ module.exports = [
     selectors: ['#demo-folder'],
   }),
 
-  // Hover states
-  //// Tile
-  scenario('State hover', {
-    actions: [hover('#test-tile-1')],
-    selectors: ['#test-tile-1'],
-  }),
-  scenario('State hover with thumbnail', {
-    actions: [hover('#test-tile-2')],
-    selectors: ['#test-tile-2'],
+  // Entire tile hover & focus states
```

Comment:
Why changing all of these? Maybe at least with separate PR...

Reply:
I changed the test images because the hover effects were not showing, they were getting clipped.

Reply:
But you also renamed and changed a lot of them. That ends up showing as fully changed tests. Therefore we can't see if this hover effect didn't break old UI.

Reply:
yeah.. I agree. Ideally would be good to firstly see the changes and then rename, if needed with separate pr

Reply:
Reverted back to original tests and test images.